.github/codecov.yaml

```diff
@@ -0,0 +1,12 @@
+coverage:
+  status:
+    project:
+      default:
+        target: 50%
+        threshold: 1%  
+    patch:
+      default:
+        target: 50%
+        threshold: 1%
+        informational: true
```

Comment:
This also ensures the patch coverages are strictly informational

Reply:
This is great!